sos_brazil/urls.py

```diff
@@ -16,4 +16,9 @@
 from django.contrib import admin
 from django.urls import include, path
 
-urlpatterns = [path("admin/", admin.site.urls), path("api/", include("ongs.urls"))]
+urlpatterns = [
+    path("admin/", admin.site.urls),
+    path("api/", include("campaigns.urls")),
+    path("api/", include("ongs.urls")),
+    # path("api/", include("users.urls")),
```

Comment:
Shouldn't it be available?

Reply:
Commented for not being implemented until the moment I tested in this branch…